cephadm-preflight-ubuntu.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: cephadm preflight on Ubuntu
+  hosts: all
+  tasks:
+    - apt:
+        upgrade: safe
+      when: ansible_facts['distribution_release']  == 'focal'
+
+    - apt_repository:
+        repo: deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ /
+        state: present 
+        filename: devel:kubic:libcontainers:stable.list
+        update_cache: no
+      when: ansible_facts['distribution_release']  == 'focal' 
+
+    - apt_key:
+        url: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/Release.key
+        state: present
+
+    - name: install apt packages
+      apt:
+        name: "{{ item }}"
+        state: present
+        update_cache: yes
+      loop:
+        - python3
+        - podman
```

Comment:
on ubuntu, I actually would recommend to use docker at this point. (especially given that the kubic repositories are deprecated by now)

Reply:
Thanks, Then, if the Ubuntu version is 20.04, It will use docker, and if the Ubuntu version is 20.10 or higher, It will use podman.